VMBackup/main/freezesnapshotter.py

```diff
@@ -261,17 +263,19 @@ def takeSnapshotFromGuest(self):
         blob_snapshot_info_array = None
         all_snapshots_failed = False
         try:
-            if self.g_fsfreeze_on :
-                run_result, run_status = self.freeze()
             if( self.para_parser.blobs == None or len(self.para_parser.blobs) == 0) :
                 run_result = CommonVariables.FailedRetryableSnapshotFailedNoNetwork
                 run_status = 'error'
                 error_msg = 'T:S taking snapshot failed as blobs are empty or none'
                 self.logger.log(error_msg, True, 'Error')
                 all_failed = True
                 all_snapshots_failed = True
+                return run_result, run_status, blob_snapshot_info_array, all_failed, all_snapshots_failed, unable_to_sleep, is_inconsistent
```

Comment:
instead of having multiple return statements, we can have "run_result == CommonVariables.success and" in next if statement.
we always do in similar fashion in our code base.

Reply:
This is like pre-check. With "or self.takeCrashConsistentSnapshot == True" also added to the conditions, it will become complex ifs. Keeping it as is.